tests/console/firewalld.pm

```diff
@@ -157,15 +157,15 @@ sub test_masquerading {
     record_info 'Masquerading tests', 'Test Rules using Masquerading';
 
     if (uses_iptables) {
-        assert_script_run("iptables -t nat -L --line-numbers");
+        script_run("iptables -t nat -L --line-numbers");
         assert_script_run("iptables -t nat -L PRE_public_allow --line-numbers | sed '/^num\\|^\$\\|^Chain/d' | wc -l | tee /tmp/nr_rules_nat_pre.txt");
         assert_script_run("iptables -t nat -L POST_public_allow --line-numbers | sed '/^num\\|^\$\\|^Chain/d' | wc -l | tee /tmp/nr_rules_nat_post.txt");
     } elsif (is_leap("<16.0") || is_sle("<16")) {
-        assert_script_run("nft list chain ip firewalld");
+        script_run("nft list table firewalld");
```

Comment:
I'm not sure if nft is used correctly here. This for example return in the test run:

```
# wait_serial expected: qr/DZotS-\d+-/u
# Result:

Error: No such file or directory
list table firewalld
           ^^^^^^^^^
DZotS-1-
```

It just does not fail because of the script_run use over assert_script_run

Reply:
Fixed now. In the future I would like to move the `grep` and `test` commands out of bash to perl.